src/main/java/org/testeditor/fixture/web/WebDriverFixture.aml

```diff
@@ -80,7 +74,7 @@ interaction type executeScript {
 
 
 interaction type checkEnabled {
-	template = "check if element " ${element} " is enabled"
+	template = "Check " ${element} " selected"
```

Comment:
Why not return a boolean value and change the template to "Is" ${element} "enabled?" and leave the check to the testeditor (not the fixture).